addons/crm/static/src/js/forecast/forecast_kanban_column_quick_create.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module alias=crm.ForecastKanbanColumnQuickCreate **/
+import Widget from 'web.Widget';
+import {_t} from 'web.core';
+ 
+/**
+ * File intended for `web` (with forecast_kanban.xml) (can be used to add next "temporal" group)
```

Comment:
Not sure this comment is really necessary (quite obvious ^^) (valable for other occurences)